integration_tests/run_tests.py

```diff
@@ -0,0 +1,91 @@
+import os
+
+MODELS = ["agent_wait_time", "zendesk_ticket_data"]
+
+
+def run_command(command: str, message: str):
+    os.system(f"echo '{message}'")
```

Comment:
this could just be a python `print` right?

Reply:
For some reason, print statements get printed on the actions console after all the system commands are finished running.

Reply:
probably would need flushing

```
print("This is my string", flush=True)
```